pyproject.toml

```diff
@@ -5,8 +5,8 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "coc.py"
 authors = [{ name = "mathsman5133" }]
-maintainers = [{ name = "majordoobie" }, { name = "MagicTheDev" }, { name = "Kuchenmampfer" }, { name = "doluk" }]
-version = "2.3.1"
+maintainers = [{ name = "majordoobie" }, { name = "MagicTheDev" }, { name = "Kuchenmampfer" }, { name = "lukasthaler" }]
```

Comment:
Why was @doluk removed lol

Reply:
Honestly I dont know
I cant find the commit